src/components/Results.tsx

```diff
@@ -31,93 +30,63 @@ export const Results: React.FC<{
     [facilities, facilityIds]
   );
 
-  const handleResults: () => Promise<LatLngTuple | undefined> = useCallback(async () => {
-    logger.info('requst results');
+  const handleResults = useCallback(async () => {
+    logger.info('Init results fetch');
     setLoading(true);
     setError(undefined);
 
     try {
       if (searchParams === undefined) {
         throw new Error('searchParams must be provided');
       }
-
-      const body = {
-        accommodation: {
-          location: {
-            lon: center[1],
-            lat: center[0],
-            radius: 20000
-          },
-          arrival: searchParams.arrival,
-          departure: searchParams.departure,
-          roomCount: searchParams.roomCount
-        },
-        passengersƒ: [
-          {
-            type: 'ADT',
-            count: searchParams.adults
-          },
-          {
-            type: 'CHD',
-            count: searchParams.children,
-            childrenAges: [13]
-          }
-        ]
-      };
-      // @todo remove mocks
-      const mocks = true;
-      if (mocks) {
-        const accommodations = mockOffers.data.accommodations;
-        Object.keys(accommodations).map((key) =>
-          dispatch({
-            type: 'SET_RECORD',
-            payload: {
-              name: 'facilities',
-              record: {
-                id: key,
-                ...accommodations[key]
-              }
+      const res = await axios.request<OffersResponse>(
+        new OffersRequest(center, searchParams)
+      );
+      if (
+        res.data === undefined ||
+        res.data.data === undefined ||
+        res.data.data.derbySoft.data === undefined
+      ) {
+        throw Error('Data undefiend');
```

Comment:
I guess we need more clear error messages. In that case, it may be `Unable to get offers request response`

Reply:
done